canonical-web-and-design/ubuntu.com
# Logo section update - part 1

## Done

- Updated the logo section with the [new Vanilla logo section pattern](https://vanillaframework.io/docs/patterns/logo-section) on:
  - homepage
  - /aws
  - /azure
  - /automotive
  - /ai/services

There will be a few of these PRs to come, there are logo sections all over the project that need updating, and would make for one massive PR otherwise.

Also - I used VSCode's auto indent tool to help me out while pasting in the many new logo image templates, it made unrelated indent adjustments in a number of places throughout the files that were updated.

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site at https://ubuntu-com-11184.demos.haus
- See that the logo sections look good
- Do the same for:
  - https://ubuntu-com-11184.demos.haus/aws
  - https://ubuntu-com-11184.demos.haus/azure
  - https://ubuntu-com-11184.demos.haus/automotive
  - https://ubuntu-com-11184.demos.haus/ai/services
- Be sure to test on mobile, tablet and desktop screen sizes

## Issue / Card

Partially fixes #11130
